lib/mobile/mobile.dart

```diff
@@ -338,8 +338,8 @@ class PlaygroundMobile {
     _router.listen();
   }
 
-  List<Element> _getTabElements(Element element) =>
-      element.querySelectorAll('a');
+//  List<Element> _getTabElements(Element element) =>
```

Comment:
Is this an unused method removal? Any reason to leave it as a comment rather than purging it compeltely?